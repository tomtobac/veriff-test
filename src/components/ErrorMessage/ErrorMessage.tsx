import Button from '../Button';

type Props = {
	onClick: VoidFunction;
};
export const ErrorMessage: React.FC<Props> = ({ onClick }) => {
	return (
		<article>
			<p>Error: Something went wrong</p>
			<Button type="button" onClick={onClick}>
				Try Again
			</Button>
		</article>
	);
};
