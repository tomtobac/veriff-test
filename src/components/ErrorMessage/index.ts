export { ErrorMessage as default } from './ErrorMessage';
