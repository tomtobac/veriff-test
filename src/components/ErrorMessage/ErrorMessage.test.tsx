import { render, screen } from '@testing-library/react';
import { ErrorMessage } from './ErrorMessage';

describe('ErrorMessage', () => {
	it('should render as it is', () => {
		const { asFragment } = render(<ErrorMessage onClick={() => {}} />);
		expect(asFragment()).toMatchSnapshot();
	});
});
