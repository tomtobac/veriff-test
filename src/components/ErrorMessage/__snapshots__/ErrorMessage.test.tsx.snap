// Vitest Snapshot v1

exports[`ErrorMessage > should render as it is 1`] = `
<DocumentFragment>
  <article>
    <p>
      Error: Something went wrong
    </p>
    <button
      class="Button"
      type="button"
    >
      Try Again
    </button>
  </article>
</DocumentFragment>
`;
